engine/src/main/java/com/arcadedb/serializer/BinarySerializer.java

```diff
@@ -332,9 +334,10 @@ else if (properties == 0)
     return null;
   }
 
-  public void serializeValue(final Database database, final Binary content, final byte type, Object value) {
+  public void serializeValue(final Database database, final Binary serialized, final byte type, Object value) {
     if (value == null)
       return;
+    Binary content = dataEncryption != null ? new Binary() : serialized;
```

Comment:
Why do you need to create a new Binary() object in the case of encryption content?